szkolenie1/SeleniumTest1.cs

```diff
@@ -13,43 +14,109 @@ namespace szkolenie1
     public class SeleniumTest1
     {
         ChromeDriver driver;
-
-        [Test]
-        public void CheckInput()
+        [SetUp]
+        public void SetUp()
         {
             driver = new ChromeDriver();
             driver.Navigate().GoToUrl("https://www.seleniumeasy.com/test/basic-first-form-demo.html");
-
             Thread.Sleep(1000);
-            driver.FindElement(By.Id("at-cv-lightbox-close")).Click();
-            driver.FindElement(By.Id("user-message")).SendKeys("Sending Keys");
-            driver.FindElement(By.ClassName("btn-default")).Click();
+            var xOnPopup = driver.FindElement(By.Id("at-cv-lightbox-close"));
+            xOnPopup.Click();
+        }
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
 
+        [Test]
+        public void CheckInput()
+        {
+            //arrange
+            var testText = "Sending Keys";
+            var enterMessage = driver.FindElement(By.Id("user-message"));
+            var showMessage = driver.FindElement(By.ClassName("btn-default"));
             var text = driver.FindElement(By.Id("display")).Text;
-
-            Assert.AreEqual("Sending Keys", text);
-            driver.Quit();
+            //act
+            enterMessage.SendKeys(testText);
+            showMessage.Click();
+            //assert
+            Assert.AreEqual(testText, text);
         }
 
         [Test]
         public void CheckSum()
         {
-            driver = new ChromeDriver();
-            driver.Navigate().GoToUrl("https://www.seleniumeasy.com/test/basic-first-form-demo.html");
+            //arrange
+            var firstDigit = "8";
+            var secondDigit = "4";
+            var sum = firstDigit + secondDigit;
+            var sum1 = driver.FindElement(By.Id("sum1"));
+            var sum2 = driver.FindElement(By.Id("sum2"));
+            var countButton = driver.FindElements(By.ClassName("btn-default"))[1];
+            var total = driver.FindElement(By.Id("displayvalue")).Text;
+            //act
+            sum1.SendKeys(firstDigit);
+            sum2.SendKeys(secondDigit);
+            countButton.Click();
+            //assert
+            Assert.AreEqual(sum, total);
```

Comment:
This won't work. total variable has value of element which doesn't have result yet. First you have to click countButton and after that you can assign value to total variable.